File: atividade_nicolas/alunos/nicolas_g2/francisco_alexandro/combustivel.js
import prompt from 'prompt-sync'
const input = prompt()

function main(){
  //entrada valor inicial
  const valor_inicial = Number(input('Valor do Litro de Gasolina: '))
  
  //calculo da composição
  let petrobras = 0.36 * valor_inicial
  let icms = 0.27 * valor_inicial
  let etanol_onidro = 0.13 * valor_inicial
  let pis_pasep = 0.1 * valor_inicial
  let distribuicao_e_revenda = 0.14 * valor_inicial
  
  //lista do valor de cada item na composição
  console.log('\n\tComposição dos preços\n')
  console.log(`Petrobras: R$ ${petrobras.toFixed(2)}`)
  console.log(`ICMS: R$ ${icms.toFixed(2)}`)
  console.log(`Etanol Onidro: R$ ${etanol_onidro.toFixed(2)}`)
  console.log(`Cide, PIS/Pasep e Cofins: R$ ${pis_pasep.toFixed(2)}`)
  console.log(`Distribuição e revenda: R$ ${distribuicao_e_revenda.toFixed(2)}\n`)
  
  //entrada do ajuste
  const porcentagem_reajuste = Number(input('Digite o valor de reajuste em %: '))
  
  //calculando ajuste
  const valor_ajustado = calcular_ajuste(valor_inicial,porcentagem_reajuste)
  
  //lista com ajuste
  petrobras = 0.36 * valor_ajustado
  icms = 0.27 * valor_ajustado
  etanol_onidro = 0.13 * valor_ajustado
  pis_pasep = 0.1 * valor_ajustado
  distribuicao_e_revenda = 0.14 * valor_ajustado
  
  console.log('\n\tComposição ajustada\n')
  console.log(`Petrobras: R$ ${petrobras.toFixed(2)}`)
  console.log(`ICMS: R$ ${icms.toFixed(2)}`)
  console.log(`Etanol Onidro: R$ ${etanol_onidro.toFixed(2)}`)
  console.log(`Cide, PIS/Pasep e Cofins: R$ ${pis_pasep.toFixed(2)}`)
  console.log(`Distribuição e revenda: R$ ${distribuicao_e_revenda.toFixed(2)}\n`)
  
}
  main()
  
  function calcular_ajuste(valor,ajuste){
    
    const gasolina = 0.73 * valor
    return valor + ((ajuste/100) * gasolina)
  }

File: atividade_nicolas/alunos/nicolas_g2/giovana_lima/combustivel.py
def main():
    valor_gasolina_posto = float(input('\tValor inicial do litro da gasolina: '))

    petrobras = 0.36*valor_gasolina_posto
    icms = 0.27*valor_gasolina_posto
    etanol = 0.13*valor_gasolina_posto
    imposto_federal = 0.10*valor_gasolina_posto
    distribuição = 0.14*valor_gasolina_posto

    print(f'\nComposição de preços da gasolina a R$ {valor_gasolina_posto:.2f}.\n')
    print(f'Petrobras: R$ {petrobras:.2f}')
    print(f'ICMS: R$ {icms:.2f}')
    print(f'Etanol onidro: R$ {etanol:.2f}')
    print(f'Cide, Pis/Pasep: R$ {imposto_federal:.2f}')
    print(f'Distribuição e revenda: R$ {distribuição:.2f}\n')

    valor_reajuste = float(
        input('\tDigite o valor do reajuste em porcentagem: '))

    reajuste = 0.36 + (valor_reajuste/100)
    valor_final = ((0.73*valor_gasolina_posto) *
                   (1 + valor_reajuste/100)) + 0.27*valor_gasolina_posto

    reajuste_petrobras = 0.36*valor_final
    petrobras = reajuste_petrobras * valor_final
    icms = 0.27*valor_final
    etanol = 0.13*valor_final
    imposto_federal = 0.10*valor_final
    distribuição = 0.14*valor_final

    print(f'\nComposição de preços da gasolina a R$ {valor_final:.2f}.\n')
    print(f'Petrobras: R$ {petrobras:.2f}')
    print(f'ICMS: R$ {icms:.2f}')
    print(f'Etanol onidro: R$ {etanol:.2f}')
    print(f'Cide, Pis/Pasep: R$ {imposto_federal:.2f}')
    print(f'Distribuição e revenda: R$ {distribuição:.2f}\n')

    print(f'Valor final da gasolina no posto: {valor_final:.2f}')


if __name__ == '__main__':
    main()
